### ISSUE TITLE
Add link to the service tier documentation

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/treecreeper#252]
Could we add a link to the service tier informational text which goes to the page which describes all the different service tiers available to choose from and what they mean?

![image](https://user-images.githubusercontent.com/1569131/82686126-3a22de80-9c4d-11ea-9ecd-0aee509947ad.png)

**Comment by wheresrhys** [comment]
Yeah, that should be possible. We already parse descriptions as markdown on the edit page, so they can contain links. We can then easily add a link to the tech hub page to the description for the service tier field.
